### DISCUSSION TITLE
Is port 4447 needed to be open to use webservices?  wsdl

### DISCUSSION THREAD

**Discussion by jpertuz-19** [Keyfactor/ejbca-ce#561] [General]
My wsdl:  https://10.10.10.10:8443/ejbca/ejbcaws/ejbcaws?wsdl

But run create new certificate: API error refused  10.10.10.10  port  4447

my netstats

tcp        0      0 127.0.0.1:9990          0.0.0.0:*               LISTEN      6235/java           
tcp        0      0 0.0.0.0:8080            0.0.0.0:*               LISTEN      6235/java           
tcp        0      0 0.0.0.0:8443            0.0.0.0:*               LISTEN      6235/java           
tcp        0      0 127.0.0.1:4447          0.0.0.0:*               LISTEN      6235/java           

The webservices tests worked with the container. 

But this server is dedicated  installed step by step

**Comment by jpertuz-19** [comment]
<interface name="management">
            <inet-address value="${jboss.bind.address.management:0.0.0.0}"/>

For Secutiry  only permit (iptables)  source IP  of my API.

**Comment by primetomas** [comment]
WS only requires the TLS port. Don't know what client you use. 
Have you tried using clientToolBox? I regularly run that over the internet with only port 8443 open in firewalls.
https://doc.primekey.com/ejbca/ejbca-operations/ejbca-operations-guide/command-line-interfaces/ejbca-client-toolbox
